asmrun/startup.c

```diff
@@ -86,7 +86,7 @@ extern value caml_start_program (void);
 extern void caml_init_ieee_floats (void);
 extern void caml_init_signals (void);
 
-#ifdef _MSC_VER
+#if defined(_MSC_VER) && __STDC_SECURE_LIB__ >= 200411L
```

Comment:
I can't find a place either for GCC or MSVC where it's explicitly documented, but an undefined token is treated as zero (this certainly works for the relevant old versions of cl)